src/main/java/org/testng/ITestContext.java

```diff
@@ -83,6 +84,12 @@
 
   List<Module> getGuiceModules(Class<? extends Module> cls);
 
+  default void addGuiceModule(Module module) {}
+
+  default List<Module> getGuiceModules() {
```

Comment:
It looks too similar to `getGuiceModules(Class<? extends Module> cls)`, what do you think about `getAllGuiceModules()`?

Reply:
Sure. Wil refactor the name to this